documentation/modules/exploit/freebsd/local/ip6_setpktopt_uaf_priv_esc.md

```diff
@@ -0,0 +1,112 @@
+## Vulnerable Application
+
+  This module exploits a race and use-after-free vulnerability in the
+  FreeBSD kernel IPv6 socket handling. A missing synchronization lock
+  in the `IPV6_2292PKTOPTIONS` option handling in `setsockopt` permits
+  racing `ip6_setpktopt` access to a freed `ip6_pktopts` struct.
+
+  This exploit overwrites the `ip6po_pktinfo` pointer of a `ip6_pktopts`
+  struct in freed memory to achieve arbitrary kernel read/write.
+
+  This module has been tested successfully on:
+
+  * FreeBSD 9.0-RELEASE #0 (amd64)
+  * FreeBSD 9.1-RELEASE #0 r243825 (amd64)
+  * FreeBSD 9.2-RELEASE #0 r255898 (amd64)
+  * FreeBSD 9.3-RELEASE #0 r268512 (amd64)
+  * FreeBSD 12.0-RELEASE r341666 (amd64)
+  * FreeBSD 12.1-RELEASE r354233 (amd64)
+
+
+## Verification Steps
+
+  1. Start `msfconsole`
+  2. Get a session
+  3. `use exploit/freebsd/local/ip6_setpktopt_uaf_priv_esc`
+  4. `set SESSION <SESSION>`
+  5. `check`
+  6. `run`
+  7. You should get a new *root* session
+
+
+## Options
+
+  ### NUM_SPRAY
+
+  Spray iterations (default (`256`)
+
+  ### NUM_SPRAY_RACE
+
+  Race iterations (default: `32`)
+
+  ### ForceExploit
+
+  Override check result (default: `false`)
+
+  ### WritableDir
+
+  A directory where we can write files (default: `/tmp`)
```

Comment:
Can you please dedent each of these sections? Especially `## Options` and `## Scenarios` where the indentation changes how it's rendered:

## Indented

![image](https://user-images.githubusercontent.com/58950994/87720490-0367d100-c783-11ea-9585-93f4acf81514.png)

## Not Indented

![image](https://user-images.githubusercontent.com/58950994/87720541-17133780-c783-11ea-8295-b6b82a7ab5ba.png)

Reply:
done